bin/alignment.py

```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+import numpy as np
+from scipy.interpolate import interp1d
+from sys import stdin
+import sys
```

Comment:
according to python imports convention , the imports must be in order of first python built-in libraries , then 3rd party modules , at last project related modules